Popt 1.19 now frees the memory returned by poptGetArg.

--- a/converter/babeltrace.c
+++ b/converter/babeltrace.c
@@ -449,7 +449,7 @@
 	do {
 		ipath = poptGetArg(pc);
 		if (ipath)
-			g_ptr_array_add(opt_input_paths, (gpointer) ipath);
+			g_ptr_array_add(opt_input_paths, (gpointer) strdup(ipath));
 	} while (ipath);
 	if (opt_input_paths->len == 0) {
 		ret = -EINVAL;
@@ -880,6 +880,10 @@
 	free(opt_output_path);
 	free(opt_debug_info_dir);
 	free(opt_debug_info_target_prefix);
+	for (i = 0; i < opt_input_paths->len; i++) {
+		char *ipath = g_ptr_array_index(opt_input_paths, i);
+		free(ipath);
+	}
 	g_ptr_array_free(opt_input_paths, TRUE);
 	if (partial_error)
 		exit(EXIT_FAILURE);
